### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by mansipadgelwar**
Make ecom responsive.

### CODE REVIEW

**Review by Kothalkarpriya — APPROVED** [review]
Code is well structured and good to merge

**Review by simran61 — APPROVED** [review]
Look good...clean code

File: src/components/Modal/ProductFilterModal.jsx [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        .filter((item) => item.checked)
+        .map((item) => item.label.toLowerCase());
+
+      if (categoriesChecked.length) {
+        filteredData = filteredData?.filter((item) =>
+          categoriesChecked.includes(item.categoryName)
+        );
+      }
+
+      //filter by price range
```
Comment (vaibhav18matere) [inline src/components/Modal/ProductFilterModal.jsx:55]: remove the comments

File: src/components/Modal/ProductFilterModal.jsx [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+                  <input
+                    className="checkbox"
+                    type="checkbox"
+                    name={item.label}
+                    id={item.id}
+                    checked={item.checked}
+                    onChange={() => handlChangeChecked(item.id)}
+                  />
+                  <label htmlFor={item.label} className="checkbox-detail">
+                    {item.label}
```
Comment (vaibhav18matere) [inline src/components/Modal/ProductFilterModal.jsx:138]: try destructuring. `item.` is getting repeated here.

**Review by vaibhav18matere — APPROVED** [review]
looks good to merge ;

**Pull request merged**